NEWS

```diff
@@ -6,6 +6,21 @@
 ##
 #
 
+## 1.2.0 (June 13, 2017)
+
+Supported platforms
+  - Shared memory: KNEM, CMA, XPMEM, SYSV, Posix
+  - RDMA interconnects with VERBs interface (InfiniBand, RoCE, iWarp, Omni-Path),
+    Cray Gemini, Cray Aries, Mellanox ConnectX4/5
+  - ISAs: x86, x86_64, ARM 64bit, Power
+Features:
+  - added support for InfiniBand DC and UD transport including native MLX5 architecture support
```

Comment:
- Added support for InfiniBand DC and UD transports, including accelerated verbs for Mellanox devices
- Full support for PGAS/SHMEM interfaces, blocking and non-blocking APIs
- Support for MPI tag matching, both in software and offload mode
- Zero copy protocols and rendezvous, registration cache
- Handling transport errors
- Flow control for DC/RC
- Dataypes support: contiguous, IOV, generic
- Multi-threading support